ofl/notoserifolduyghur/METADATA.pb

```diff
@@ -0,0 +1,22 @@
+name: "Noto Serif Old Uyghur"
+designer: "Google"
+license: "OFL"
+category: "SERIF"
+date_added: "2022-10-11"
+fonts {
+  name: "Noto Serif Old Uyghur"
+  style: "normal"
+  weight: 400
+  filename: "NotoSerifOldUyghur-Regular.ttf"
+  post_script_name: "NotoSerifOldUyghur-Regular"
+  full_name: "Noto Serif Old Uyghur Regular"
+  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/old-uyghur)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+source {
+  repository_url: "https://github.com/notofonts/old-uyghur.git"
+  archive_url: "https://github.com/notofonts/old-uyghur/releases/download/NotoSerifOldUyghur-v1.002/NotoSerifOldUyghur-v1.002.zip"
+}
+is_noto: true
```

Comment:
@simoncozens missing subset, languages and primary script.

Reply:
That suggests we have no lang data for this script. I'll see what I can do.